src/Application/Redux/StateUtils.js

```diff
@@ -0,0 +1,157 @@
+import ImmutableAsset from './../Models/Asset';
+import { List } from 'immutable';
+import _ from 'underscore';
+
+const PRICE_INCREASE_PERCENTAGE = 7;
+const BASE_CLICK_INCOME = 1;
+
+const createNews = (news, newArticle) => {
+  const copyOfNews = news.slice();
+  copyOfNews.push(newArticle);
+  if (copyOfNews.length > 10) copyOfNews.shift();
```

Comment:
we tend never to put if blocks on the same line, that's pure pref tho i'm not sure there's any real argument there